names.txt

```diff
@@ -16,5 +16,5 @@ Kelechi Joanna Ojelabi
 Austin Wopara
 Awosise Oluwaseun
 Adetayo Akinsanya Samuel
-Segun Samuel
-Adedolapo Olutuyo
+Austin Wopara
```

Comment:
Remove duplicate names

Reply:
You sent two messages. Which of them am I to follow